api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -23,6 +23,21 @@ const (
 	ActionRelocate = DRAction("Relocate")
 )
 
+// DRPCType is the type of the DRPC based on the application handle
+type DRPCType string
+
+// These are the valid values for DRPCType
+const (
+	// Managed is the DRPC type for the applications that are managed by OCM
+	Managed = DRPCType("Managed")
+
+	// Unmanaged is the DRPC type for the applications that are not managed by OCM
+	Unmanaged = DRPCType("Discovered")
+
+	// Invalid is the DRPC type for the applications that are not recognized by Ramen
+	Invalid = DRPCType("Invalid")
```

Comment:
Where will you ever have an `Invalid` workload type? I would just get rid of it.